app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextConverter.java

```diff
@@ -122,7 +126,8 @@
             YamlFrontMatterExtension.create(),
             TypographicExtension.create(),        // https://github.com/vsch/flexmark-java/wiki/Typographic-Extension
             GitLabExtension.create(),             // https://github.com/vsch/flexmark-java/wiki/Extensions#gitlab-flavoured-markdown
-            FootnoteExtension.create()            // https://github.com/vsch/flexmark-java/wiki/Footnotes-Extension#overview
+            FootnoteExtension.create(),           // https://github.com/vsch/flexmark-java/wiki/Footnotes-Extension#overview
+            AdmonitionExtension.create()
```

Comment:
suggestion: insert above footnote (=1 change), and add the link to the extension documentation

Reply:
applied